CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `opentelemetry-instrumentation-tornado` Tornado: Conditionally create SERVER spans
   ([#867](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/889))
 
+- `opentelemetry-instrumentation-starlette` Adding unit tests for conditional server span creation
+  ([#897](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/897))
+
```

Comment:
```suggestion
```

Reply:
No changelog entry is needed, this is not a public facing change, will add the corresponding label so that tests pass :+1:

Reply:
removed entry from changelog 👍